### ISSUE TITLE
Sonarr thinks episode is downloaded, it is not.

### ISSUE THREAD

**Issue by TheDutchMC** [Sonarr/Sonarr#4520]
**Describe the bug**

Sonarr thinks episode(s) are downloaded, when they are not.

**To Reproduce**

Might be just my pure unluck with the torrent, but.
1. Download Lucifer S05
2. Watch sonarr grab a torrent that does not the just-releases part two of the season
3. Sonarr thinks the entire season is downloaded, where only half really is.

**Expected behavior**

Sonarr realizes that half the season is missing, and goes searching for it.

**Screenshots**

Sonarr:
![image](https://user-images.githubusercontent.com/28569170/120379425-07abc700-c320-11eb-8d66-700169a7859c.png)

Filesystem:
![image](https://user-images.githubusercontent.com/28569170/120379483-1c885a80-c320-11eb-9f6c-fb4b80987ccc.png)

**Platform Information (please complete the following information):**
 - OS:  Ubuntu 20.04
 - Docker:  Yes, running under Kubernetes
 - .net Framework (Windows) or mono (macOS/Linux) (System -> Status):  Mono, or whatever may be inside the Docker container
 - Browser and Version (Only needed for UI issues): 
 - Sonarr Version:  3.0.6.1196
 - Sonarr Branch:  N,A

**Trace Logs**
Turn on Trace logs under Settings -> General and wait for the bug to occur again.
**Upload the full log file here (or another site (e.g. pastebin) and link it). Issues will be closed, if they do not include this!**

I cant manually mark the file as not-downloaded, so it's nearly impossible to reproduce this on my instance.

**Comment by thezoggy** [comment]
what was the release sonarr grab, season pack?  s05 was split into two parts.. so a season pack for s05 may be incomplete..

**Comment by TheDutchMC** [comment]
Correct, it grabbed a season pack. So it is perfectly reasonable for that to be incomplete.

However the issue is that Sonarr does not seem to check if it is complete, but just assumes it is. Which is fine in most cases, but in this case, it is not.

So my proposed solution would be to add a check to actually check what was downloaded, rather than blindly assuming everything was.

**Comment by TheDutchMC** [comment]
Interesting to note as well, pressing 'Refresh & Scan' does not pick it up either.

**Comment by Taloth** [comment]
Check the episode history of episode 15. And check the episode file path specified in the episode details.
